source/plugins/github.downloader/index.js

```diff
@@ -0,0 +1,106 @@
+import axios from 'axios';
+
+let studio = null;
+
+let githubdownloader = {
+	downloading: false,
+	progress: {started: false},
+	/**
+	 * Show a list for tutorials from a github repository
+	 * 
+	 * @param {String} repository - username/repository
+	 * @returns A list with all the tutorials
+	 */
+	async getTutorials(repository) {
```

Comment:
Then github plugin should be a generic one that allows downloading files. `getTutorials` should be implemented using this github downloader in the tutorials plugin.